spaceship/lib/spaceship/portal/ui/select_team.rb

```diff
@@ -31,6 +31,20 @@ class UserInterface
       #     {...}
       #   ]
 
+      def self.ci?
+        if Object.const_defined?("FastlaneCore")
+          return FastlaneCore::Helper.ci?
+        end
+        return false
+      end
+
+      def self.interactive?
+        if Object.const_defined?("FastlaneCore")
```

Comment:
Just to be save, maybe this should be
```ruby
if Object.const_defined?("FastlaneCore") && FastlaneCore.const_defined?("Helper")
```
Because maybe someone defines a `FastlaneCore` module, without actually importing `fastlane_core`